google-cloud-spanner/lib/google/cloud/spanner/backup/job.rb

```diff
@@ -32,6 +35,10 @@ class Backup
         # @see https://cloud.google.com/spanner/reference/rpc/google.longrunning#google.longrunning.Operation
         #   Long-running Operation
         #
+        # @deprecated Use the long-running operation returned by
+        # {Google::Cloud::Spanner::Admin::Database#database_admin.create_backup}
```

Comment:
I'm not very sure about the format here. But we can change it to the way it's written in [Overview](https://github.com/googleapis/google-cloud-ruby/pull/13157/files#diff-05fa1d482ea992b3ac65b78a0caa8461aa93e7952fb0461902cb6cf58b9b554dR26) file.

Reply:
I updated it to the way we specified in the overview.